simpl_eeg/eeg_objects.py

```diff
@@ -69,7 +69,7 @@ def generate_epochs(self, tmin, tmax, start_second):
             tmin=tmin,
             tmax=tmax,
             preload=True,
-            baseline=(0, 0)
+            baseline=(0, 0) # THIS is what's causing the reading at 0 seconds to be completely white
```

Comment:
I think I needed to set it to that for one of the other plots to work. I'll try running it and see if it breaks anything.